diff --git a/dist/transpilers/swc.js b/dist/transpilers/swc.js
index eeddd4fbb2f18a7e8b20bb628d5f5c806ae20594..36fa1ae673acae42a1a15d3095c047fdf4f2d47d 100644
--- a/dist/transpilers/swc.js
+++ b/dist/transpilers/swc.js
@@ -138,7 +138,7 @@ function createSwcOptions(compilerOptions, nodeModuleEmitKind, swcInstance, swcD
     //   if not alwaysStrict, will emit implicitly whenever module target is non-ES *and* transformed module syntax is emitted.
     // For node, best option is to assume that all scripts are modules (commonjs or esm) and thus should get tsc's implicit strict behavior.
     // Always set strictMode, *unless* alwaysStrict is disabled and noImplicitUseStrict is enabled
-    const strictMode = 
+    const strictMode =
     // if `alwaysStrict` is disabled, remembering that `strict` defaults `alwaysStrict` to true
     (alwaysStrict === false || (alwaysStrict !== true && strict !== true)) &&
         // if noImplicitUseStrict is enabled
@@ -165,7 +165,7 @@ function createSwcOptions(compilerOptions, nodeModuleEmitKind, swcInstance, swcD
                     ignoreDynamic: nodeModuleEmitKind === 'nodecjs',
                 }
                 : undefined,
-            swcrc: false,
+            swcrc: true,
             jsc: {
                 externalHelpers: importHelpers,
                 parser: {